pest/src/iterators/pairs.rs

```diff
@@ -249,14 +249,7 @@ impl<'i, R: RuleType> Pairs<'i, R> {
         let (prev_line, prev_col) = (self.cursor.line, self.cursor.col);
 
         let part = &input[self.cursor.end..end];
-        let (l, c) = position::line_col(part, part.len());
-
-        // Because the `original_line_col` returns (line, col) is start from 1
-        let l = l - 1;
-        let mut c = c - 1;
-        if c < 1 {
-            c = 1
-        }
+        let (l, c) = position::original_line_col(part, part.len(), (0, 0));
```

Comment:
Can this still use the line_col() as before (so that it could benefit from fast-line-col when it's enabled)?

Reply:
If I do that, I will change the line_col to add a argument, is this method is private?

Reply:
line_col() is pub(crate), so it should be only exposed within the crate (in other words, changing it shouldn't be a semver-breaking change)